llarp/rpc/lokid_rpc_client.cpp

```diff
@@ -215,6 +215,36 @@ namespace llarp
       return ftr.get();
     }
 
+    void
+    LokidRpcClient::LookupLNSNameHash(
+        dht::Key_t namehash, std::function<void(std::optional<std::string>)> resultHandler)
+    {
+      LogDebug("Looking Up LNS NameHash ", namehash);
+      const nlohmann::json req{{"type", 2}, {"name_hash", namehash.ToHex()}};
+      Request(
+          "rpc.lns_resolve",
+          [r = m_Router, resultHandler](bool success, std::vector<std::string> data) {
+            std::optional<std::string> result = std::nullopt;
+            if (success)
+            {
+              try
+              {
+                const auto j = nlohmann::json::parse(data[1]);
+                const auto itr = j.find("encrypted_value");
+                if (itr != j.end())
+                {
+                  result = lokimq::from_hex(itr->get<std::string>());
+                }
+              }
+              catch (...)
```

Comment:
Should have a comment with a `catch (...) do nothing` block, to save others and future-you from figuring out what you're ignoring and why.